workflow/rules/common.smk

```diff
@@ -1591,3 +1607,17 @@ def get_delly_excluded_regions():
         )
     else:
         return []
+
+
+def get_alignment_props():
+    def inner(wildcards):
+        if is_activated("custom_alignment_properties"):
+            alignment_prop_column = config["custom_alignment_properties"]["column"]
+            prop_name = extract_unique_sample_column_value(
+                wildcards.sample, alignment_prop_column
+            )
+            if pd.notna(prop_name):
+                return custom_alignment_props.loc[prop_name, "path"]
+        return "results/alignment-properties/{group}/{sample}.json"
+
+    return inner
```

Comment:
I thought it was the preferred way to not pass wildcards as param. How do we device when to use an inner function and when not? In assume in case no element of the function access the wildcards directly we can use an inner function?